devtools/release/find_block_60_days_ago.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
```

Comment:
Could you let this script update the file [util/constant/src/sync.rs](https://github.com/nervosnetwork/ckb/pull/4254/files#diff-de161884740a17e0b012a104898c602a2284c34f3d33eca7b76fe6602047b8fb) directly?